tests/integration/commands/test_deletion.py

```diff
@@ -0,0 +1,52 @@
+import confluent_kafka
+import pytest
+from click.testing import CliRunner
+
+from esque.cli.commands import delete_topic
+
+
+@pytest.fixture()
+def basic_topic(num_partitions, topic_factory):
+    yield from topic_factory(num_partitions, "basic-topic")
+
+
+@pytest.fixture()
```

Comment:
There are also collisions with underscore and period, we could test that too 
https://jaceklaskowski.gitbooks.io/apache-kafka/kafka-topics.html

Reply:
Yeah I know. Tried that. But kafka doesn't even let me create two colliding topics.